### DISCUSSION TITLE
Keep versioning same as vuepress

### DISCUSSION THREAD

**Discussion by b00f** [vuepress-theme-hope/vuepress-theme-hope#598] [Q&A] [answered]
I suggest to keep versioning same as vuepress (at least Major and Minor numbers) 
 For example, if Vuepress version is `1.8.2`, Vuepress-theme-hope keeps its version to `1.8.x`.

This will help a lot for better maintenance.

**Comment by Mister-Hope** [comment, marked as answer]
Nope, first, I am actually breaking the versioning strategy to align the major version.

I am supposed to raise minor version with new features, and major version with breaking changes.

Now, both of them are raised with minor version, while bug fixes is published as patch.

In your opinion, I cannot release any features or breaking changes unless vueprees publish another minor version, that's absolutely unreasonable. Making breaking changes or big features with patch version is unreasonable either.

Besides, the theme doesn't relay on the VuePress versions. VuePress is not changing its API options. So the theme should work with all vuepress versions.
